<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSyntroDBClass</class>
 <widget class="QMainWindow" name="CSyntroDBClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>553</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SyntroDB</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>553</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Actions</string>
    </property>
    <addaction name="actionBasicSetup"/>
    <addaction name="actionConfiguration"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure ...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionBasicSetup">
   <property name="text">
    <string>Basic setup</string>
   </property>
  </action>
  <action name="actionConfiguration">
   <property name="text">
    <string>Configuration</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
